tests/test_base_repository.py

```diff
@@ -0,0 +1,19 @@
+from unittest import IsolatedAsyncioTestCase
+
+from spyplane.database.config_repository import ConfigRepository
+from spyplane.spy_plane import bot
+
+
+class BaseRepositoryTests(IsolatedAsyncioTestCase):
+
+    async def asyncSetUp(self) -> None:
```

Comment:
Missing documentation

Reply:
What is your expected documentation here? Example?

Reply:
These are unit tests so no doc needed sorry.